fitbenchmarking/fitbenchmarking
# Added a more descriptive error msg when default problem set is missing

## Description of work

Fixes #1213 




## Testing Instructions



1. Run `pytest fitbenchmarking/cli/tests/test_main.py` to verify that test pass.
2. Delete the Average difficulty NIST problem sets and run `fitbenchmarking` to verify the error msg.


## Checklist
